docs/user-guide/creating-application/eso/aws-eso.md

```diff
@@ -2,7 +2,7 @@
 
 To add secrets from **AWS Secrets Manager**, we need to create a generic Kubernetes secret for AWS authentication.
 
-Create a Kubernetes secret in the namespace in which the application is to be deployed using base64 encoded AWS access-key and secret-access-key. You can use devtron generic chart for this.
+Create a Kubernetes secret in the namespace in which the application is to be deployed using base64 encoded AWS access-key and secret-access-key. You can use Devtron generic chart for this.
```

Comment:
Create a Kubernetes secret in the namespace in which the application is to be deployed using base64 encoded AWS access-key and secret-access-key. You can use a Devtron generic chart for to achieve it.

Reply:
You can use a Devtron generic chart _for to_ achieve it. ?

Reply:
@Shubham9t9, I have changed the sentence.